### PR TITLE
adds Tuesday outage alert banner to site

### LINKED ISSUE #5312: Actions for Open Library Outage: Tuesday, June 15, 2021 @ ~8:30am PT
# Situation 
Our Internet Archive operations team alerts us:

An `<energy provider>` Planned Outage Coordinator alerts us:
- a burned electrical pole in front of the `<redacted>` datacenter is being replaced on Tuesday
- this will require deenergizing the feed line and we will lose power at `<redacted>` for the duration of the event.  This duration is highly dependent on the specifics of how the job goes, which they won't know until they start it (e.g. if they can extract the burned pole without damaging the base, etc)

tl;dr We should expect to lose power at `<redacted>` for at a minimum several hours starting at or after 8:30am Pacific this coming Tuesday June 15.

cc: @jonah-archive, @samuel-archive, @cdrini, @bfalling, @jamesachamp 

# Resolution

Let's please make sure:
1. [x] our site-wide banner is applied to the website for 2021-06-14 (**Monday**) and
2. [x] a tweet is scheduled morning for 8am Pacific, giving a full 24h

twitter com_home (5)

### PR DESCRIPTION
**Pull request by mekarpeles**
Closes #5312

### Screenshot

To be added after on testing.openlibrary.org

### CODE REVIEW

File: openlibrary/templates/site/body.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
       </div>
       $# Paste next line into announcement variable to show the blue banner
       $# <strong>Title:</strong> Description &nbsp; <a href="#" data-ol-link-track="AnnouncementsBar|BlogWidget" class="btn primary">Learn More</a>
-      $ announcement = Starting 8:30am PT, Tuesday, June 15th Open Library will be <strong>offline</strong> for maintenance
+      $ announcement = 'Starting 15:30 UTC, Tuesday, June 15th Open Library will be <strong>offline</strong> for maintenance'
```
Comment (mekarpeles) [inline openlibrary/templates/site/body.html:29]: ty!

**Comment by jamesachamp** [comment]
Added inline styling to announcement banner in order to avoid a CSS rebuild during patch deployment.

**Comment by cdrini** [comment]
Re patch deployed :+1:

File: openlibrary/templates/site/body.html [hunk 2]
```diff
@@ -26,9 +26,9 @@
       </div>
       $# Paste next line into announcement variable to show the blue banner
       $# <strong>Title:</strong> Description &nbsp; <a href="#" data-ol-link-track="AnnouncementsBar|BlogWidget" class="btn primary">Learn More</a>
-      $ announcement = None
+      $ announcement = 'Starting 15:30 UTC, Tuesday, June 15th Open Library will be <strong>offline</strong> for maintenance'
       $if announcement:
-        <div class="page-banner page-banner-body">
+        <div class="page-banner page-banner-body" style="display: block;">
```
Comment (mekarpeles) [inline openlibrary/templates/site/body.html:31]: Thank you!

**Comment by cdrini** [comment]
Bottle of rum

**Pull request closed without merging**